src/api/app/models/project.rb

```diff
@@ -28,6 +28,9 @@ class Project < ApplicationRecord
     @commit_opts = {}
   end
 
+  after_save :update_staging_workflow
+  after_destroy :update_staging_workflow
```

Comment:
what's the point of storing anything after destroy? If the project is gone, the staging workflow is too

Reply:
@coolo, this `after_destroy` only affects `staging_projects`.

Reply:
I see